src/core/configProvider/configContext.ts

```diff
@@ -31,6 +31,8 @@ export type Config = {
       props: React.AnchorHTMLAttributes<HTMLAnchorElement>,
     ) => JSX.Element;
     Head?: (props: { children: React.ReactNode }) => JSX.Element;
+    EventCardContent?: (props: any) => JSX.Element;
```

Comment:
Unexpected any. Could it be `CardProps`? Is the real issue in rollup-package?

Reply:
no, actually it should be the event item...